packages/vscode-extension/src/utilities/utils.ts

```diff
@@ -24,7 +25,10 @@ export class Utils implements UtilsInterface {
     try {
       keybindingsJsonPath = path.join(
         homedir(),
-        "Library/Application Support/Code/User/keybindings.json"
+        Platform.select({
+          macos: "Library/Application Support/Code/User/keybindings.json",
+          windows: "AppData\\Roaming\\Code\\User\\keybindings.json",
```

Comment:
Still breaks on Cursor :c

Reply:
We can deduce used IDE by checking value of `vscode.env.appName`, so now it also works on Cursor c: